src/Exchange/GhostNode.f90

```diff
@@ -129,14 +129,14 @@ subroutine gnc_df(this, m1, m2)
     call this%read_data()
     !
     ! -- Trap for implicit gnc but models are in different solutions
-    if(this%m1%idsoln /= this%m2%idsoln) then
-      if(this%implicit) then
-        write(errmsg, '(a)') 'Error.  GNC is implicit but models are in ' //   &
-                             'different solutions.'
+    if (this%m1%idsoln /= this%m2%idsoln) then
+      if (this%implicit) then
+        write (errmsg, '(a)') 'Error.  GNC is implicit but models are in '// &
```

Comment:
Not sure if it is possible to add a space before `//`. If it is, do we want to do this?

Reply:
I'm looking for a way to do this, a space would seems like it would be more consistent with the configuration as it currently stands.